pkg/nodegraph/Library/NodegraphApp.js

```diff
@@ -9,7 +9,7 @@
 import {App} from '../../Library/App/App.js';
 import {LocalStoragePersistor} from '../../Library/App/storage.js';
 import {RecipeService} from '../../Library/App/RecipeService.js';
-import {StoreUpdateService} from '../../Library/NodeGraph/StoreUpdateService.js';
+import {StoreService} from '../../Library/NodeGraph/StoreService.js';
 import {DevToolsService} from '../../Library/DevTools/DevToolsService.js';
```

Comment:
I'm don't have 100% recall on this, but I believe it's dealt with at Worker level.

Reply:
I'm referring to DevToolsService ... it may not be fully plumbed yet, but I think it's intended to be automatic.

Reply:
same here